tests/certification/secretstores/azure/keyvault/managed-identity-app/Dockerfile

```diff
@@ -4,5 +4,5 @@ WORKDIR /app
 RUN apt-get update && apt-get install wget curl --yes
 RUN wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash
 RUN dapr init --slim
-ENV AzureKeyVaultName=dapr2-conf-test-kv
+ENV AzureKeyVaultName=dapr3-conf-test-kv
```

Comment:
Is this also leading to any failures?

Reply:
Not that I see any!

Reply:
This is no longer used.. I will be removing this test.